packages/nimble-components/src/table/tests/table.spec.ts

```diff
@@ -844,4 +848,47 @@ describe('Table', () => {
             expect(table.validity.duplicateRecordId).toBeTrue();
         });
     });
+
+    describe('with validatable columns', () => {
```

Comment:
I think there should also be a test that verifies that the value of `validity.invalidColumnConfiguration` gets updated when the set of slotted columns updates. I think that case is handled in the code (I assume that's why `this.tableValidator.validateColumnConfigurations(this.columns);` is called within `validate()` on the Table), so it would be good to have a test for that.

Reply:
Added a test.